src/domain/collaboration/callout/useCallouts/useCallouts.ts

```diff
@@ -176,6 +177,7 @@ const useCallouts = (params: UseCalloutsParams): UseCalloutsProvided => {
           draft,
           editable,
           flowStates,
+          group: displayLocationTagset?.tags[0],
```

Comment:
I think here you can do the mapping to a `CalloutDisplayLocation` and then you won't need to be dragging that `CalloutDisplayLocationValuesMap` everywhere, this would be the only place where you do the mapping from `HOME_0` to `CalloutDisplayLocation.HomeTop / 'HOME_TOP'`

Correct me if I'm wrong

Reply:
Adding a util function for extracting the display location from the tagset as CalloutDisplayLocation type.